File: AMS.API/DTO/BatchMaster/SearchBatchMasterRequestParams.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AMS.API.DTO.BatchMaster
{
    public class SearchBatchMasterRequestParams
    {
        public string BatchId { get; set; }

        public string BatchName { get; set; }

        #region All other details

        public int SchemeId { get; set; }

        //public string SchemeName { get; set; }

        public int JobRoleId { get; set; }

        public int CityId { get; set; }

        public int VTP_Id { get; set; }

        public int TotalCandidates { get; set; }

        #endregion
    }
}

File: AMS.Web/src/app/models/batch-master/search-batch-master-request-params.ts
export class SearchBatchMasterRequestParams 
{
    BatchId: string = null;
    BatchName: string = null;

    //#region All other details

    SchemeId: number = 0;
    //SchemeName: number = 0;
    JobRoleId: number = 0;
    CityId: number = 0;    
    VTP_Id: number = 0;
    TotalCandidates: number = 0;

    //#endregion

    IsActive: boolean = true;
}